### ISSUE TITLE
strange behavior of lastUnit

### ISSUE THREAD

**Issue by ibrezac** [stubbornella/oocss#43]
Hello!

The following is an example that demonstrates unexpected behavior of .lastUnit on FF 3.6 and Chrome 6.0.  IE8 does not appear to be affected.  I have not tried other browsers yet.  

<div class="line">  
   <div class="unit size1of2">
      stuff here
   </div>
   <div class="unit size1of2 lastUnit">
      <div style="width: 50px; float: right;">
        This div will cause additional space added at the bottom of the parent div (lastUnit)
     </div>
   </div>
</div>

Removing clear: both from .lastUnit:after appears to fix the issue, but I am not sure if this created other problems.

Any ideas?

Thanks!
-igor

**Comment by stubbornella** [comment]
do you have an example online somewhere? I think github is stripping some of the code.

**Comment by ibrezac** [comment]
Yep, sorry about that.  Here it is: http://www.ypass.net/lastUnit.htm
